native/cocos/platform/java/modules/SystemWindow.cpp

```diff
@@ -57,11 +57,15 @@ void SystemWindow::setWindowHandle(void *handle) {
 #if (CC_PLATFORM == CC_PLATFORM_ANDROID)
     //The getWindowHandle interface may have been called earlier, causing _handleMutex to be occupied all the time.
     bool lockSuccess = _handleMutex.try_lock();
-    bool needNotify = _windowHandle == nullptr;
     _windowHandle = handle;
-    if (needNotify) {
+    if (!lockSuccess) {
```

Comment:
In multi-window handling logic, CocosSurfaceView creates multiple windows and involves multithreaded interactions.